.github/workflows/ci.yml

```diff
@@ -69,7 +69,12 @@ jobs:
 
       - name: Test with pytest
         run: |
-          uv run pytest tests/ --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
+          uv run pytest tests/test_auth.py::test_token_refresh --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
+          uv run pytest tests/test_auth.py::test_nexus_client_reloads_tokens --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
+          uv run pytest tests/test_auth.py::test_nexus_client_reloads_domain --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
+          uv run pytest tests/test_auth.py::test_token_refresh_expired --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
+
+          uv run pytest tests/ -v --ignore=tests/test_auth.py --doctest-modules --cov=qnexus --cov-report=html --cov-report=term
```

Comment:
I think you need to include the `--cov-append` [flag](https://pytest-cov.readthedocs.io/en/latest/config.html#reference) so the coverage report is not overwritten with each call to `pytest`